Alignment/TrackerAlignment/plugins/CosmicRateAnalyzer.cc

```diff
@@ -315,33 +315,35 @@ CosmicRateAnalyzer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSe
           else if (SiStripDetId::TOB == subdetId1)
           {
              ++nhitinTOB;
-             TOBDetId tobId1(detId1);
           }
 //////////////////////////////////////////////////////////////////////////////////////////////////
 //			Hit information in TIB		                                	//
 /////////////////////////////////////////////////////////////////////////////////////////////////
           else if (SiStripDetId::TIB == subdetId1)
           { 
              ++nhitinTIB;
-             TIBDetId tibId1(detId1);
           }
 //////////////////////////////////////////////////////////////////////////////////////////////////
 //			Hit information in TID		                                	//
 /////////////////////////////////////////////////////////////////////////////////////////////////
           else if (SiStripDetId::TID == subdetId1)
           { 
              ++nhitinTID;
-             TIDDetId tidId1(detId1);
-             if (tidId1.isZMinusSide())
+	 
+			 //tTopo->...something...(...detId1...);
+             //TIDDetId tidId1(detId1); //from previous code
+             tTopo->tidIsZMinusSide(detId1);
```

Comment:
other question (github doesn't show me): did you manage to solve the space/tab issue?